.github/workflows/rust-build.yml

```diff
@@ -120,12 +116,12 @@ jobs:
       additional_args: ${{ inputs.additional_args }}
       custom_cargo_commands: ${{ inputs.custom_cargo_commands }}
       login_private_registry: ${{ inputs.publish_public_registry != 'true' }}
-      skip-test-publish: ${{ inputs.skip-test-publish }}
+      test-publish-required: ${{ inputs.publish_docker == 'true' && inputs.publish_binary == 'true' && inputs.publish_binary == 'true' }}
```

Comment:
When are these flags set in the real world? Shouldn't that only be true for runs on the main branch and not already as part of a PR? Also am I missing something or is the same field checked twice here?

Reply:
Currently we used them here https://github.com/search?q=org%3AForesightMiningSoftwareCorporation+skip-test-publish++NOT+repo%3AForesightMiningSoftwareCorporation%2Fgithub&type=code but I agree that we should maybe not make them available to enforce at org level

Reply:
Thanks for the typo find, I've fixed it